code/modules/genetics/recipes/mutation_recipe_defines.dm

```diff
@@ -56,7 +56,7 @@
 					compare_string = compare_string + "M~"
 			else
 				compare_string = compare_string + "M~"
-
+		log_debug("initialize_mutation_recipes(): Loaded recipe [example_recipe] with string [compare_string]")
```

Comment:
Just double-check that the log is supposed to stay.

Reply:
Yeah we want that there to make sure that recipe strings are loading like they ought to